docs/README.md

```diff
@@ -1,37 +1,43 @@
 # The OPA Website and Documentation
+The content and tooling is separated into a few places:
 
-This directory contains all of the Markdown, HTML, Sass/CSS, and other assets needed
-to build the [openpolicyagent.org](https://openpolicyagent.org) website. See the
-section below for steps to build the site and test documentation changes
-locally.
+[devel/](./devel/) - Developer documentation for OPA (not part of the website)
 
-The raw OPA documentation can be found under the [content/docs](./content/docs)
-directory.
 
-> ### Developing OPA
-> For documentation on developing OPA, see the [devel](./devel) directory.
+[website/](./website/) - This directory contains all of the Markdown, HTML, Sass/CSS,
+and other assets needed to build the [openpolicyagent.org](https://openpolicyagent.org)
+website. See the section below for steps to build the site and test documentation changes
+locally. This content is not versioned for each release, it is common scaffolding for
+the website.
+
+[content/](./content/) - The raw OPA documentation can be found under the 
+directory. This content is versioned for each release and should have all images
+and code snippets alongside the markdown content files.
 
 ## Run the site locally
 
 You can run the site locally [with Docker](#run-the-site-locally-using-docker) or
-[without Docker](#run-the-site-locally-without-docker). Regardless of your method,
-you'll need install [npm](https://www.npmjs.com/get-npm).
+[without Docker](#run-the-site-locally-without-docker).
 
-### Run the site locally using Docker
+### Generating Versioned Content ###
 
-To run the site locally using [Docker](https://docker.com), first install the
-necessary static assets using npm:
+> This *MUST* be done before you can serve the site locally!
 
-```bash
-npm install
-```
+The site will show all versions of doc content from the tagged versions listed
+in [RELEASES](./RELEASES).
 
-Generate some required files:
-```bash
-make load-docs
+To generate them run:
+```shell
+make generate
 ```
+The content then will be placed into `docs/generated/docs/$VERSION/`.
```

Comment:
The content seems to be placed in `docs/website/generated` not `docs/generated`.

Reply:
Will update.